### PR TITLE
Plugin for showing fortune quotes

### PR DESCRIPTION
**Pull request by hpcodecraft**
A little plugin to show a random fortune quote.

Fortune Plugin Screenshot

Clicking "next cookie" will refresh the plugin.
Needs fortune installed via Homebrew.

### CODE REVIEW

**Comment by hpcodecraft** [comment]
I looked into Travis' output because of the failing checks and tried to implement the changes the checks suggested, but then the plugin stopped working. Are green checks a hard requirement?

**Comment by tresni** [comment]
Yes, I'll take a look and see if I can give you some guidance.

File: Lifestyle/fortune.5m.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+fortune="/usr/local/bin/fortune"
+
+echo "|dropdown=false image=$cookie"
+echo "---"
+
+if [[ -x $fortune ]]; then
+  echo "Next cookie|refresh=true image=$cookie"
+  echo "---"
+  echo " |trim=false"
+  echo "$($fortune|/usr/bin/sed 's/-/–/g; s/^/    /; s/$/  |font=Avenir size=14 trim=false/')"
```
Comment (tresni) [inline Lifestyle/fortune.5m.sh]: changing this line to the following worked for me:

```sh
$fortune|/usr/bin/sed 's/-/–/g; s/^/    /; s/$/  |font=Avenir size=14 trim=false/'
```

This passes all tests.  if it doesn't work can you tell me what you are getting/seeing?

Comment (hpcodecraft) [inline Lifestyle/fortune.5m.sh]: Thanks for your help, your version works!

**Pull request merged**